### ISSUE TITLE
error when visiting the page

### ISSUE THREAD

**Issue by nathany** [campoy/golang-groups#4]
Hey @campoy 

> You are not authorized to make that request API requests must be key-signed, oauth-signed, or accompanied by a key: https://www.meetup.com/meetup_api/docs/#authentication

https://go-meetups.appspot.com/

**Comment by campoy** [comment]
Yeah, Meetup.com broke my app and I don't have time to fix it right now :-/
